docs/pages/integrations/okta/apis/get_logs.mdx

```diff
@@ -0,0 +1,44 @@
+import { Callout } from "nextra/components";
+
+# Get Logs (Okta)
+
+Retrieve a list of system log events events from Okta.
+
+<Callout type="info">
+	For more information on the API for listing system log events, see [List all
+	System Log
+	Events](https://developer.okta.com/docs/api/openapi/okta-management/management/tag/SystemLog/#tag/SystemLog/operation/listLogEvents).
+</Callout>
+
+**SDK Import:**
+
+```python
+from admyral.actions import get_okta_logs
+```
+
+## Arguments:
+
+| Argument Name               | Description                                                                               | Required |
+| --------------------------- | ----------------------------------------------------------------------------------------- | :------: |
+| **Query** `query`           | Filter log events by specified keywords.                                                  |    -     |
```

Comment:
Could you add an example query here?

Reply:
I was thinking more about the example from our password policy monitoring workflow: `policy.lifecycle.update Password`